apps/api/v1/pages/api/teams/[teamId]/_auth-middleware.ts

```diff
@@ -41,7 +41,11 @@ export async function canUserAccessTeamWithRole(
   /** If not ADMIN then we check if the actual user belongs to team and matches the required role */
   if (!isSystemWideAdmin) args.where = { ...args.where, members: { some: { userId, role } } };
   const team = await prisma.team.findFirst(args);
-  if (!team) throw new HttpError({ statusCode: 401, message: `Unauthorized: ${role.toString()} required` });
```

Comment:
The error message was thrown like this `Unauthorized: [Object object] required`, which is why this was changed